.gitignore

```diff
@@ -10,3 +10,6 @@ vendor
 /docs/
 /site/content/en/docs/Reference/agc*
 /dist/
+*.zip
+/packages/cdk/lib/wes_adapter/package
+/packages/cdk/lib/wes_adapter/wes_adapter
```

Comment:
Can these files redirected to a sub-folder? e.g. "/packages/cdk/lib/wes_adapter/dist/..."

Reply:
the 'package' sub is supposed to be the dist folder. I can move wes_adapter.zip back in there to keep all artifacts in the same place.